src/main/webapp/app/buildagent/build-agent-summary/build-agent-summary.component.spec.ts

```
@@ -131,36 +130,33 @@ describe('BuildAgentSummaryComponent', () => {
         alertService = TestBed.inject(AlertService);
         modalService = TestBed.inject(NgbModal);
         alertServiceAddAlertStub = jest.spyOn(alertService, 'addAlert');
+
+        websocketSubject = new Subject<BuildAgentInformation[]>();
+        mockWebsocketService.subscribe.mockReturnValue(websocketSubject.asObservable());
     }));
 
     beforeEach(() => {
         jest.clearAllMocks();
+        mockWebsocketService.subscribe.mockReturnValue(websocketSubject.asObservable());
     });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)